docs/guides/cert-manager.md

```diff
@@ -0,0 +1,55 @@
+# Install cert-manager
+
+Install the CustomResourceDefinition resources separately.
```

Comment:
This is great! Could we also just add a link to their official install guide in case this becomes outdated?